app/views/layouts/_analytics.html.erb

```diff
@@ -1,17 +1,20 @@
 <!-- Google Analytics -->
 <script>
+/* global ga, window */
```

Comment:
since this is refactoring the way we instanciate the GA JS, could we pull this out into a separate PR to split it out from the event tracking changes?

Reply:
Good point - I'll cherry pick and put in a separate pull request.